chain/epoch-manager/src/tests/mod.rs

```diff
@@ -2276,69 +2275,6 @@ fn test_protocol_version_switch() {
     );
 }
 
-#[test]
-fn test_protocol_version_switch_with_shard_layout_change() {
```

Comment:
From the test name it seems like we want to keep this test, no? Won't claim I understand the test though.